claims.go

```diff
@@ -37,7 +37,7 @@ func (c StandardClaims) Valid() error {
 	// default value in Go, let's not fail the verification for them.
 	if c.VerifyExpiresAt(now, false) == false {
 		delta := time.Unix(now, 0).Sub(time.Unix(c.ExpiresAt, 0))
-		vErr.Inner = fmt.Errorf("token is expired by %v", delta)
+		vErr.Inner = fmt.Errorf("Token is expired by %v", delta)
```

Comment:
That's true.  I'm not sure if whether to consider this an API breaking change, though.  Even though it's not advisable, some users do string-compare these errors in application logic.